src/app/users/users-profile/users-profile.component.html

```diff
@@ -9,8 +9,8 @@
     <span>{{userDetail.name}}</span>
     <mat-icon class="margin-lr-5" *ngIf="userDetail.gender" svgIcon="{{userDetail.gender}}"></mat-icon>
     <span class="toolbar-fill"></span>
-    <button mat-button class="margin-lr-3" *ngIf="user.name === urlName || user.isUserAdmin" i18n [routerLink]="['../../update/', urlName]"><mat-icon>mode_edit</mat-icon><span>Edit Profile</span></button>
-    <a mat-button class="margin-lr-3" *ngIf="user.name === urlName" i18n [planetChangePassword]="userDetail">Change password</a>
+    <button mat-raised-button color="accent" class="margin-lr-3" *ngIf="user.name === urlName || user.isUserAdmin" i18n [routerLink]="['../../update/', urlName]"><mat-icon>mode_edit</mat-icon><span>Edit Profile</span></button>
```

Comment:
these changes are not related to this PR looks like you created branch from another branch. Please make sure you always create branch from **master** only